variables.tf

```
@@ -86,8 +87,26 @@ variable "fifo_topic" {
   default     = false
 }
 
+variable "fifo_queue" {
```

```suggestion
variable "fifo_queue_enabled" {
```

Our convention/pattern

---

the same for `fifo_topic_enabled`

---

Since fifo_topic is the same attribute as aws_sns_topic, shouldn't we keep that consistent ?

---

let's keep it